core/src/main/java/ai/timefold/solver/core/impl/phase/scope/AbstractStepScope.java

```diff
@@ -44,10 +46,18 @@ public void setBestScoreImproved(Boolean bestScoreImproved) {
         this.bestScoreImproved = bestScoreImproved;
     }
 
+    public void incrementMoveEvaluationCount(Move<?> move) {
+        if (getPhaseScope().isEnableCollectMetrics()) {
```

Comment:
Let's change the name; 'isMetricCollectionEnabled()', better English.
Also, it would be nice if the method were documented - why do we ever need to disable metrics?